arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -0,0 +1,1670 @@
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted the request to deactivate the device."
+          },
+          "204": {
+            "description": "Successfully deactivated the device."
+          }
+        },
+        "deprecated": false,
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.StorSimple/managers/{managerName}/devices/{deviceName}/publicEncryptionKey": {
+      "post": {
+        "tags": [
+          "Managers"
+        ],
+        "description": "Returns the public encryption key of the device.",
+        "operationId": "Managers_GetDevicePublicEncryptionKey",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "deviceName",
+            "in": "path",
+            "description": "The device name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/managerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The public key.",
+            "schema": {
+              "$ref": "#/definitions/PublicKey"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.StorSimple/managers/{managerName}/encryptionSettings/default": {
+      "get": {
+        "tags": [
+          "Managers"
+        ],
+        "description": "Returns the encryption settings of the manager.",
+        "operationId": "Managers_GetEncryptionSettings",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/managerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The encryption settings.",
+            "schema": {
+              "$ref": "#/definitions/EncryptionSettings"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.StorSimple/managers/{managerName}/extendedInformation/vaultExtendedInfo": {
```

Comment:
Per your email you said there are no plans to add any more values, but note that if in the future you need to parameterize this it will be a breaking change.  @amarzavery are there any other concerns?